.travis.yml

```diff
@@ -1,18 +1,11 @@
 language: ruby
+
 rvm:
-  - 1.9.3
-  - 2.0.0
-  - 2.1.0
-  - jruby
+  - 2.2.4
+  - 2.3.0
 
 gemfile:
-  - test/gemfiles/Gemfile.rails-3.2.x
```

Comment:
Let's maintain support for ruby 1.9.3 and rails 3.2 if it is reasonably easy to do so. See today's https://github.com/binarylogic/authlogic/pull/497 which adds the following CI matrix exclusions:

```diff
+    - rvm: 2.2.5
+      gemfile: test/gemfiles/Gemfile.rails-3.2.x
+    - rvm: 2.3.1
+      gemfile: test/gemfiles/Gemfile.rails-3.2.x
```

Hopefully, the only change this PR will need to make here is the addition of a rails 5.0 gemfile. *fingers crossed*